mufasa/README.md

```diff
@@ -22,32 +14,104 @@ unpack that file in your makeflow file.
 - All desired output files must be packed into `output.tar.gz`. This is the file put in the outbox or error directory in the event of an error.
 
 
+## Basic Usage
+---------
+The general structure for configuring Mufasa is shown in the src/main.py file.
+The point of this file is to create a program that can be run like the following:
+```
+python src/main.py -i /path/to/inbox -o /path/to/outbox -m /path/to/makeflow/file -e /path/to/error
+```
+This will monitor start monitoring the inbox directory for input files. 
+When they are processed, they will be moved to the outbox directory if successful, if not, then they are moved to the error directory.
+
+***IMPORTANT NOTE***: You should ***NOT*** `cp` files into the inbox directory. Instead, you must `mv` them.
+
+This program performs the following basic operations. 
+The first step is to create an instance of the Workflow() class.
+```
+wf = Workflow(path_to_makeflow_file, name_of_expected_inputfile)
+```
+The `path_to_makeflow_file` is the absolute path to the makefile which must be in the same directory as the source code necessary for it to run.
+The expected input file should be the name of the tarfile that makefile unpacks to extract the other input files.
+
+Then you must create an instance of a WorkflowScheduler.
+The first three arguments of a WorkflowScheduler specify the path to the outbox directory, the path to the created Workflow, and the path to the error directory.
+The two other options are `total_limits` and `workflow_limits` which both have default values specified in the WorkflowScheduler.py module, but it is highly recommended to override them.
+The total resource limits can be specified as a dictionary and should be passed as an argument to the WorkflowScheduler(). 
+They must be a dictionary similar to the following:
+```
+total_limits = {
```

Comment:
For consistency with Makeflow and other parts of the cctools, call this `cores`, `memory`, `disk`.
Make sure to give the units here, otherwise it's not clear: do memory and disk refer to MB, GB, or something else?